src/rest/Query.ts

```diff
@@ -232,4 +233,9 @@ export interface IQuery {
    * This parameter is normally controlled by {@link SearchInterface.options.allowEmptyQuery} option.
    */
   allowQueriesWithoutKeywords?: boolean;
+
+  /**
+   * Specifies an array of request to retrieve facet values for the MLFacet component
+   */
+  facets?: IFacetRequest[];
```

Comment:
Would `facetRequests` be a better name here?

Reply:
I preferred having the same name on request/response like we see for the category facet. I won't change these now as it's on the API as well